spec/system/viewing_user_menu_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+RSpec.describe "Viewing User Menu", system: true do
+  fab!(:user) { Fabricate(:user) }
+
+  let(:user_menu) { PageObjects::Components::UserMenu.new }
+
+  describe "when viewing replies notifications tab" do
+    fab!(:topic) { Fabricate(:topic) }
+
+    it "should display group mentioned notifications in the tab" do
+      Jobs.run_immediately!
+
+      mentionable_group = Fabricate(:group, mentionable_level: Group::ALIAS_LEVELS[:everyone])
+      user_in_mentionable_group = Fabricate(:user).tap { |user| mentionable_group.add(user) }
+
+      _post_with_group_mention =
+        PostCreator.create!(user, topic_id: topic.id, raw: "Hello @#{mentionable_group.name}")
```

Comment:
If it is not used anywhere, do we need to assign it?

Reply:
Yup the assignment is mainly for readability.